RegistrationLib/__init__.py

```diff
@@ -3,7 +3,11 @@
 from Landmarks import *
 from RegistrationState import *
 from RegistrationPlugin import *
-from AffinePlugin import *
-from ThinPlatePlugin import *
-from LocalBRAINSFitPlugin import *
-from LocalSimpleITKPlugin import *
+
+for plugin in [
+  'Affine',
+  'ThinPlate',
+  'LocalBRAINSFit',
+  'LocalSimpleITK'
+  ]:
+  registerRegistrationPlugin('RegistrationLib.%sPlugin' % plugin)
```

Comment:
Yes, separating the two is good.  One other thing to note is that the plugin pattern is also used in the Editor and DICOM modules, so I'd like to see them all stay consistent.  I'm not opposed to a generic facility but to be honest the importlib methods look pretty clunky to my eye, while the existing pattern is a few extra lines but pretty understandable.